pymc3/linear_models/linear.py

```diff
@@ -87,7 +87,7 @@ def from_formula(cls, formula, data, priors=None, vars=None, name='', model=None
                    priors=priors, vars=vars, name=name, model=model)
 
 
-class Glm(LinearComponent):
+class GLM(LinearComponent):
```

Comment:
Yes, and furthermore we've already adopted the convention with `MAP` and `NUTS`.